chrome-2023-experiment-overview.md

```diff
@@ -0,0 +1,101 @@
+# Chrome 2023 Experiment Overview
+
+In early 2023, Google Chrome will be running an origin trial to allow developers to try out several new enhancements related to speculation rules and prefetching. These enhancements are focused around allowing the browser to automatically assist with prefetching with minimal developer work, and allowing developers to measure when their pages are successfully prefetched.
+
+## Opting into the origin trial
+
+Developers can sign up for an [origin trial](https://developer.chrome.com/docs/web-platform/origin-trials/) token.
+
+> **Note**
+> This trial is not yet available. We'll update this with a link when it launches.
+
+Most developers will only need a token for their own origin, and can use it the usual way.
+
+```http
+Origin-Trial: [token issued to your origin]
+```
+
+```html
+<meta http-equiv="origin-trial" content="[token issued to your origin]">
+```
+
+For this trial, Chrome will also accept a third-party token issued to another origin, as long as either:
+
+* an external script loaded from that origin adds it, or
+* the server delivers the document response with a `Speculation-Rules` HTTP response header which loads speculation rules and an `Origin-Trial` HTTP response header issued to that origin
+
+```http
+Origin-Trial: [third-party token for https://rules-provider.example]
+Speculation-Rules: "https://rules-provider.example/speculationrules.json"
+```
+
+## HTTP-fetched speculation rules
+
+Some developers prefer to deliver speculation rules out of line. You can do this using the `Speculation-Rules` response header.
+
+```http
+Speculation-Rules: "/speculationrules.json"
+```
+
+This will be fetched and cached like any other subresource, though at a lower priority since it doesn't block the current page load. This resource must use the correct MIME type and, if it is cross-origin, pass a [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) check.
+
+```http
+Content-Type: application/speculationrules+json
+Access-Control-Allow-Origin: *
+```
+
+## Automatic link finding
+
+With this enhancement, the browser can automatically react find links in your page which the user is likely to click. The developer simply needs to communicate which links are appropriate to prefetch, since some servers might exhibit side effects for `GET` requests.
```

Comment:
"react"?

```suggestion
With this enhancement, the browser can automatically prefetch links in your page which the user is likely to click. The developer simply needs to communicate which links are appropriate to prefetch, since some servers might exhibit side effects for `GET` requests.
```

Include prerender if that is included.